Homework_13/person.py

```diff
@@ -0,0 +1,45 @@
+from class_my_exception import PersonNameException, PersonAgeException
+
+
+class Person:
+    """
+    Class создающий персону с возможностью добавления в архив!
+    """
+
+    def __init__(self, first_name, last_name, age):
+        if len(first_name) < 10:
+            self.first_name = first_name
+        else:
+            raise PersonNameException
```

Comment:
Упрощайте запись. Чтобы не использовать лишние конструкции.
```suggestion
        if len(first_name) >= 10:
            raise PersonNameException
            
        self.first_name = first_name            
```